ui/src/app/components/runtime-configuration-panel/customize-panel.tsx

```diff
@@ -216,6 +218,31 @@ export const CustomizePanel = ({
                   )
                 }
               />
+            </FlexRow>
+          </FlexColumn>
+          <FlexColumn>
```

Comment:
do we want to show this in environments where there is only 1 zone option?

Reply:
Good idea